pkg/api/norman/customization/project/project_actions.go

```diff
@@ -9,24 +9,27 @@ import (
 	"strings"
 	"time"
 
-	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
-	"github.com/rancher/rancher/pkg/controllers/management/imported"
-
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 	"github.com/rancher/norman/api/handler"
 	"github.com/rancher/norman/httperror"
 	"github.com/rancher/norman/types"
 	"github.com/rancher/norman/types/convert"
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
 	client "github.com/rancher/rancher/pkg/client/generated/management/v3"
 	"github.com/rancher/rancher/pkg/clustermanager"
+	"github.com/rancher/rancher/pkg/controllers/management/imported"
 	"github.com/rancher/rancher/pkg/generated/compose"
+	ctrlv1 "github.com/rancher/rancher/pkg/generated/controllers/provisioning.cattle.io/v1"
```

Comment:
A lot of other places use `provisioningcontrollers` or `provcontrollers` when importing this package, or even `provisioningv1`

Reply:
renamed to `provisioningv1`